include/deal.II/matrix_free/fe_evaluation.h

```diff
@@ -4388,15 +4413,17 @@ FEEvaluationBase<dim, n_components_, Number, is_face>::
   const internal::MatrixFreeFunctions::DoFInfo::DoFAccessIndex ind =
     is_face ? dof_access_index :
               internal::MatrixFreeFunctions::DoFInfo::dof_access_cell;
+  const unsigned int n_lanes = mask.count();
 
   const std::vector<unsigned int> &dof_indices_cont =
     dof_info->dof_indices_contiguous[ind];
 
   // Simple case: We have contiguous storage, so we can simply copy out the
   // data
   if (dof_info->index_storage_variants[ind][cell] ==
-      internal::MatrixFreeFunctions::DoFInfo::IndexStorageVariants::
-        interleaved_contiguous)
+        internal::MatrixFreeFunctions::DoFInfo::IndexStorageVariants::
+          interleaved_contiguous &&
+      n_lanes == VectorizedArray<Number>::n_array_elements)
```

Comment:
n_lanes differs from n_array_elements whenever the user supplies a mask which does not have all bits set to true. This is for example the case in local time stepping algorithms. Unfortunately, it is not easy to provide a simple example since LTS is complex and the finite element formulation we use is also complex.